railties/lib/rails/rack/logger.rb

```diff
@@ -7,6 +7,9 @@
 module Rails
   module Rack
     # Sets log tags, logs the request, calls the app, and flushes the logs.
+    #
+    # Log tags can be an Array containing: methods that the <tt>request</tt> object responds to, a Proc
```

Comment:
Looking at the code below, "Log tags" in this context could be called `+taggers+` right?